.github/workflows/sims.yml

```diff
@@ -2,10 +2,8 @@ name: Sims
 # Sims workflow runs multiple types of simulations (nondeterminism, import-export, after-import, multi-seed-short)
 # This workflow will run on all Pull Requests, if a .go, .mod or .sum file have been changed
 on:
-  pull_request:
-  push:
-    branches:
-      - master
+  schedule:
```

Comment:
we should probably also run on release

Reply:
Agree

Reply:
I can run on rcs, but tags may be too late right?

Reply:
let's just figure out what makes sense from this: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows

Reply:
i think we want `release` `published` and `created`

Reply:
Alternatively, we can run them on all PRs targetting `release/v*` (and even make them required there, if possible). Releases are always tagged from those branches.